sympy/integrals/tests/test_integrals.py

```diff
@@ -1475,3 +1475,9 @@ def test_issue_4311():
         (x**4/4 - 9*x**2/2, x <= -3),
         (-x**4/4 + 9*x**2/2 - S(81)/2, x <= 3),
         (x**4/4 - 9*x**2/2, True))
+
+def test_issue_15494():
+    x = symbols('x', real=True, positive=True)
+    integrand = (exp(x/2)-2*exp(1.6*x)+exp(x))*exp(x)
+    sol = integrate(integrand, x)
+    assert str(sol) == str(Rational(2, 3)*exp(Rational(3,2)*x) + Rational(1, 2)*exp(2*x) -  0.769230769230769*exp(2.6*x))
```

Comment:
Rather than comparing the strings maybe you could compare the two solutions using floating point like:
```python
assert abs(sol - expected) < 1e-15 * exp(2.6*x)
```